pzsd_bot/model.py

```diff
@@ -23,8 +24,9 @@
         server_default=text("gen_random_uuid()"),
     ),
     Column("name", Text, nullable=False, unique=True),
-    Column("discord_snowflake", Text, nullable=False, unique=True),
+    Column("discord_snowflake", Text, nullable=True, unique=True),
```

Comment:
Be careful with unique columns that are also nullable databases are inconsistent in handling.

Doesn't matter here but I've run into issues where code blows up because select one or none where field = None returns multiple rows

Reply:
I looked it up and postgres treats each Null value as unique and distinct from one another. Apparently you'd need to  explicitly set `NULLS NOT DISTINCT` for this to not be true I think. I didn't try to test looking up a user by a null discord snowflake, but I assume it would return no rows?